src/Tizen.Wearable.CircularUI.Forms.Renderer/CirclePageRenderer.cs

```diff
@@ -270,11 +270,20 @@ void UpdateBackground()
         }
         void UpdateActionButton(bool initialize)
         {
+            //remove previous action button
+            if (_actionButton != null)
```

Comment:
Why remove previuse button?

Reply:
Fix not updating issue when replace old ActionButtonitem to  new ActionButtonItem.

Reply:
removed this code.